examples/ECR Data Mart Creation.ipynb

```diff
@@ -210,9 +210,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "myenv",
+   "display_name": "Python 3.9.13 64-bit",
```

Comment:
I think you can get rid of these lines that cause some thrashing between machines in source control by running:
```
pip install nb-clean
nb-clean clean "examples/ECR Data Mart Creation.ipynb"
```

This is good fodder for the doc changes I'm working on too - I'll try to add this for posterity.

Reply:
Done